2024/calcite-react/demo/.eslintrc.cjs

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  root: true,
+  env: { browser: true, es2020: true },
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:react-hooks/recommended',
+  ],
+  ignorePatterns: ['dist', '.eslintrc.cjs'],
```

Comment:
(optional) `'.eslintrc.cjs'` might be ignored by eslint automatically, so this is redundant

Reply:
Done